### PR TITLE
Add generic vis type

### PR DESCRIPTION
**Pull request by dvsilviyageo**
Summary: 
* Generic vis type is added to EvisualizationType.
* The Onboarding messages is empty for generic vis initially.
* It is added for all the 3 visualizations.

Screenshots:

https://user-images.githubusercontent.com/104566246/217775815-b64d3625-5f19-4a69-b6d4-8a1832347499.mp4

### CODE REVIEW

File: packages/echarts/src/index.ts [hunk 1]
```diff
@@ -81,10 +81,9 @@ export const generateBasicAnnotations = (
       onboardingMessages = heatmapFactory(contextKey, chart, coords, visElement)
       break
 
-    default:
-      throw new Error(
-        `No onboarding for visualization type ${visType} available.`
-      )
```
Comment (dvdanielamoitzi) [inline packages/echarts/src/index.ts:87]: we should actually keep this default and additionally add e.g. "You can use "Generic" to initialize an empty visAhoi instance"

File: packages/plotly/src/index.ts [hunk 2]
```diff
@@ -85,9 +85,9 @@ export const generateBasicAnnotations = (
       onboardingMessages = heatmapFactory(contextKey, chart, coords, visElement)
       break
 
-      throw new Error(
-        `No onboarding for visualization type ${visType} available.`
-      )
```
Comment (dvdanielamoitzi) [inline packages/plotly/src/index.ts:90]: same here

File: packages/vega/src/index.ts [hunk 3]
```diff
@@ -94,10 +94,9 @@ export const generateBasicAnnotations = async (
       onboardingMessages = scatterplotFactory(contextKey, vegaSpec, d3Data, visElement)
       break
 
-    default:
-      throw new Error(
-        `No onboarding for visualization type ${visType} available.`
-      )
```
Comment (dvdanielamoitzi) [inline packages/vega/src/index.ts:100]: same here

**Review by dvdanielamoitzi — CHANGES_REQUESTED** [review]
Could you please add an example demo for this vis type? maybe add a boxplot that initializes a "Generic" vis type with custom messages

File: packages/echarts/src/index.ts (hunk 1 above)
Comment (dvsilviyageo) [reply packages/echarts/src/index.ts:87]: Added default message also  'No onboarding for visualization type ${visType} available. Please use visualization type 'GENERIC' to create your own onboardings.'

File: packages/vega/src/index.ts (hunk 3 above)
Comment (dvsilviyageo) [reply packages/vega/src/index.ts:100]: Added default message also 'No onboarding for visualization type ${visType} available. Please use visualization type 'GENERIC' to create your own onboardings.'

**Review by dvdanielamoitzi — APPROVED** [review]
Thanks! This is a great enhancement! :+1:

**Pull request merged**